Sources/Widgets/Widgets3D/SplineWidget/state.js

```diff
@@ -11,7 +11,7 @@ export default function generateState() {
     .addField({ name: 'splineClose', initialValue: true })
     .addField({
       name: 'splineBoundaryCondition',
-      initialValue: boundaryCondition.DEFAULT,
+      initialValue: BoundaryCondition.DEFAULT,
```

Comment:
Where is DEFAULT defined ?

Reply:
I updated the constant file (it was indeed missing).